tests/test_procdockerstatsd.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import logging
```

Comment:
These imports are unused. Remove?

Hint: please check the new "LGTM" alerts that we have added to the repo :)

Reply:
Oh yes. Saw that. This is a cool feature:)